frontend/packages/knative-plugin/src/components/add/EventSource.tsx

```diff
@@ -112,7 +112,10 @@ export const EventSource: React.FC<Props> = ({
       apiVersion: sinkApiVersion,
       kind: sinkKind,
       name: sinkName,
-      key: craftResourceKey(sinkName, { kind: sinkKind, apiVersion: sinkGroupVersionKind }),
+      key: craftResourceKey(sinkName, {
+        kind: sinkKind,
+        apiVersion: `${sinkGroup}/${sinkVersion}`,
+      }),
```

Comment:
just curious what went wrong with `sinkGroupVersionKind`?

Reply:
craftResourcekey expects the apiVersion in `group/version` format and we were passing it in `group~version~kind` format.